### PR TITLE
teardown() can close database connections now

### PR DESCRIPTION
**Pull request by YongchangLiu98**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def tearDown(self):
         """ This runs after each test """
         db.session.remove()
         db.drop_all()
+        db.get_engine().dispose()
```
Comment (ryanjcohen) [inline tests/test_models.py:53]: wondering if we should also be doing this in test_routes.py

Comment (YongchangLiu98) [reply tests/test_models.py:53]: oh sure

Comment (YongchangLiu98) [reply tests/test_models.py:53]: > 

Done.

**Review by ryanjcohen — APPROVED** [review]

**Comment by ryanjcohen** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/dfa5f370-4610-4cc3-b00a-cc377cfff178?env_id=ibm:yp:us-south "customers-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/bcf906cc-f45b-4b30-a096-6ea68fc0ead1 "customers") deployed [customers](https://github.com/devops-customers-squad/customers/releases/tag/deploy-ibm-yp-us-south-rjc584-nyu.edu-dev-20211204-005505 "deploy-ibm-yp-us-south-rjc584-nyu.edu-dev-20211204-005505") to [dev](https://console.bluemix.net/apps/57207d43-2a44-4c6a-916f-347eb68fb4cc?env_id=ibm:yp:us-south "ibm:yp:us-south:rjc584+@nyu.edu:dev"), including commit 3ba8c110082e052086eb411aabafcc6057dd3b72

**Pull request merged**